samples/animate/anitest_vc8.vcproj

```diff
@@ -827,7 +827,7 @@
 			UniqueIdentifier="{67DA6AB6-F800-4c08-8B7A-83BB121AAD01}"
 			>
 			<File
-				RelativePath="..\sample.rc"
+				RelativePath="..\..\samples\sample.rc"
```

Comment:
I had changes in `sample.rc` path like this when I rebaked too. 

This does not look correct to me?

Reply:
This change shouldn't have been here, of course, but it's not wrong per se (both paths refer to the same file and the project should still work fine), it's just that I've clearly forgotten to rebake this one in/after 5341bf5b60 (Remove unnecessary win32-res tags from samples bakefiles, 2021-10-20).